config.go

```diff
@@ -535,7 +535,17 @@ func loadConfig() (*config, []string, error) {
 	// All data is specific to a network, so namespacing the data directory
 	// means each individual piece of serialized data does not have to
 	// worry about changing names per network and such.
+	//
+	// Check for old versions of testnet directories here since the network
+	// specific DataDir will be used after this.
 	cfg.DataDir = cleanAndExpandPath(cfg.DataDir)
+	var oldTestNet bool
+	oldDataDir := filepath.Join(cfg.DataDir, "testnet")
```

Comment:
I was just being lazy and figured I would do that after the next testnet day but you are probably correct and I will do it now.